### ISSUE TITLE
Timepicker automatically close when update state inside `onTimeChange` event.

### ISSUE THREAD

**Issue by umair-khanzada** [ecmadao/react-times#88]
Picker automatically close when I update state inside `onTimeChange` if I comment *setState()* it is not update time, below is my code please have a look.

JS: 
```
//default state.
this.state = {start: {time: '11:01', meridiem: 'AM'}};

onTimeChange({hour, minute, meridiem}) {
    console.log("hour", hour)
    console.log("minute", minute)
    console.log("meridiem", meridiem)
    this.setState({start: {time: `${hour}:${minute}`, meridiem}})
  }
```
JSX:
```      
<TimePicker timeMode="12" 
  time={this.state.start.time} meridiem={this.state.start.meridiem}  
  closeOnOutsideClick={false} onTimeChange={this.onTimeChange.bind(this)} />
```

**Comment by ecmadao** [comment]
@umair-khanzada 
Thanks for feedback! Yes there seems had a small mistake for `focused` state update, and I have just published v3.1.4 to fix it.

**Comment by umair-khanzada** [comment]
@ecmadao 
Thanks 👍
